### ISSUE TITLE
Nightwatch fails: Error connecting to chrome on port 9515

### ISSUE THREAD

**Issue by gitressa** [thinktandem/drupal-contributions#55]
This happens in both `9.x` (Note: First do #54) and `10.x`:

```
$ lando nightwatch tests/Drupal/Nightwatch/Tests/exampleTest.js
yarn run v1.22.17
$ cross-env BABEL_ENV=development node -r dotenv-safe/config -r @babel/register ./node_modules/.bin/nightwatch --config ./tests/Drupal/Nightwatch/nightwatch.conf.js tests/Drupal/Nightwatch/Tests/exampleTest.js

[Tests/Example Test] Test Suite
===============================
⠙ Connecting to chrome on port 9515...
    POST http://chrome:9515 /session - ECONNRESET
Error: socket hang up
    at connResetException (node:internal/errors:691:14)
    at Socket.socketOnEnd (node:_http_client:466:23)
    at Socket.emit (node:events:532:35)
    at endReadableNT (node:internal/streams/readable:1346:12)
⚠ Error connecting to chrome on port 9515.
_________________________________________________

TEST FAILURE: 1 error during execution; 0 tests failed, 0 passed (348ms)

 ✖ Tests/exampleTest
   An error occurred while retrieving a new session: "socket hang up"
Error: An error occurred while retrieving a new session: "socket hang up"
    at HttpRequest.emit (node:events:520:28)
    at ClientRequest.emit (node:events:532:35)
    at Socket.socketOnEnd (node:_http_client:466:9)
    at Socket.emit (node:events:532:35)
    at endReadableNT (node:internal/streams/readable:1346:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)

  Error: An error occurred while retrieving a new session: "socket hang up"
       at HttpRequest.emit (node:events:520:28)
       at ClientRequest.emit (node:events:532:35)
       at Socket.socketOnEnd (node:_http_client:466:9)
       at Socket.emit (node:events:532:35)
       at endReadableNT (node:internal/streams/readable:1346:12)
       at processTicksAndRejections (node:internal/process/task_queues:83:21)

   SKIPPED:
   - Test page
   - Page objects test page

error Command failed with exit code 5.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by gitressa** [comment]
This was fixed in #56, and Nightwatch now works, thanks @murilohpucci :+1: 

```
$ lando nightwatch tests/Drupal/Nightwatch/Tests/exampleTest.js
yarn run v1.22.18
$ cross-env BABEL_ENV=development node -r dotenv-safe/config -r @babel/register ./node_modules/.bin/nightwatch --config ./tests/Drupal/Nightwatch/nightwatch.conf.js tests/Drupal/Nightwatch/Tests/exampleTest.js

[Tests/Example Test] Test Suite
===============================
ℹ Connected to chrome on port 9515 (702ms).
  Using: chrome (98.0.4758.102) on Linux platform.

✔ Running Test page:

✔ Element <body> was visible after 27 milliseconds.
✔ Testing if element <body> contains text 'Test page text' (38ms)

OK. 2 assertions passed. (335ms)
✔ Running Page objects test page:

✔ Element <body> was visible after 20 milliseconds.
✔ Testing if element <Element [name=@body]> contains text 'Test page text' (31ms)
✔ Ensuring no deprecation errors have been triggered (8ms)

OK. 3 assertions passed. (119ms)

OK. 5  total assertions passed (4.975s)
Done in 14.04s.

```

**Comment by gitressa** [comment]
I just saw the Drupal 10.0.x-dev issue [Remove Chromedriver as a JavaScript dependency](https://www.drupal.org/project/drupal/issues/3317879), I am not sure if it affects https://github.com/lando/drupal-contributions/ and the Nightwatch configuration needs to be updated?

See also the Change record [Chromedriver has been removed as a JavaScript dependency](https://www.drupal.org/node/3319135).
